app/gui2/src/components/CircularMenu.vue

```diff
@@ -27,37 +29,42 @@ const emit = defineEmits<{
 }>()
 
 const showColorPicker = ref(false)
+
+function openDocs(url: string) {
+  window.open(url, '_blank')
+}
 </script>
 
 <template>
-  <div class="CircularMenu" :class="{ partial: !props.isFullMenuVisible }">
+  <div class="CircularMenu">
     <div
       v-if="!showColorPicker"
       class="circle menu"
       :class="`${props.isFullMenuVisible ? 'full' : 'partial'}`"
     >
-      <div v-if="!isFullMenuVisible" class="More" @pointerdown.stop="emit('openFullMenu')"></div>
-      <SvgButton
-        v-if="isFullMenuVisible"
-        name="comment"
-        class="slot2"
-        title="Comment"
-        @click.stop="emit('startEditingComment')"
-      />
-      <SvgButton
-        v-if="isFullMenuVisible"
-        name="paint_palette"
-        class="slot3"
-        title="Color"
-        @click.stop="showColorPicker = true"
-      />
-      <SvgButton
-        v-if="isFullMenuVisible"
-        name="trash2"
-        class="slot4"
-        title="Delete"
-        @click.stop="emit('delete')"
-      />
+      <template v-if="isFullMenuVisible">
+        <SvgButton
+          v-if="documentationUrl"
+          name="help"
+          class="slot1"
+          :title="`Open Documentation (${graphBindings.bindings.openDocumentation.humanReadable})`"
```

Comment:
We could also add binding for other elements (`delete component` and maybe more)